includes/attendee/attendee-repository.php

```diff
@@ -86,47 +86,77 @@ public function remove_attendee( int $event_id, int $user_id ): void {
 	/**
 	 * @throws Exception
 	 */
-	public function get_attendee( int $event_id, int $user_id ): ?Attendee {
+	public function get_attendee_for_event_for_user( int $event_id, int $user_id ): ?Attendee {
+		$attendees = $this->get_attendees_for_events_for_user( array( $event_id ), $user_id );
+		if ( empty( $attendees ) ) {
```

Comment:
Would this be too risky?
```suggestion
		if ( 1 !== count( $attendees ) ) {
```